src/main/java/hu/oe/nik/szfmv/automatedcar/Main.java

```diff
@@ -1,5 +1,6 @@
 package hu.oe.nik.szfmv.automatedcar;
 
+import hu.oe.nik.szfmv.automatedcar.model.AutomatedCar;
```

Comment:
Ez később lehet ütközést fog okozni. Majd a master mergnél ezekre figyelni kell.

Reply:
Igazából mainben majd csak WorldManager lesz, tehát ez egy felesleges módosítás, szerintem csak azért került be, hogy ne dobjon errort

Reply:
Direkt került át, az én értelmezésemben ez egy modell. De ha másként látjátok, meggyőzhető vagyok.

Reply:
Bár lehet célszerű lenne egy külön Car modell, mert az NPC-knek nem fog kelleni VirtualFunctionBus ha jól sejtem.

Reply:
Az AutomatedCar a vezérelt autó, ebbe kerül a hajtáslánc, a szenzorok stb. Része a világnak, méghozzá mozgásra képes része. Már majdnem filozófiai kérdés, hogy akkor ez most egy modell-e egy ilyen szoftverben, de nem szokott oda kerülni. Ez tipikusan egy olyan módosítás, amit más csapatokkal is célszerű egyeztetni, ha átteszitek.

A Car modell viszont mindenhonnan egy logikus absztrakciónak tűnik.